parts/k8s/kubernetescustomscript.sh

```diff
@@ -119,8 +134,15 @@ function installEtcd() {
     fi
 
     /opt/azure/containers/mountetcd.sh || exit $ERR_ETCD_VOL_MOUNT_FAIL
-    systemctl_restart 10 5 30 etcd || exit $ERR_ETCD_START_TIMEOUT
-    MEMBER="$(sudo etcdctl member list | grep -E ${MASTER_VM_NAME} | cut -d':' -f 1)"
```

Comment:
Wrapping this inside retry (see below) to deal with observed transient flakiness